packages/config/src/config/_common.js

```diff
@@ -12,8 +12,7 @@ export default () => ({
   mode: 'universal',
   modern: undefined,
 
-  // Globals
-  globalName: 'nuxt',
+  globalName: undefined,
```

Comment:
Can't we keep it here but using template literal to avoid escape?

Reply:
We can, but it will be re-assign in options.js anyway